### PR TITLE
Adding custom properties as part of TelemetryContext

### PR DESCRIPTION
**Pull request by hectorhdzg**
User can specify custom properties for all events as part of this

### CODE REVIEW

File: vNext/extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts [hunk 1]
```diff
@@ -114,6 +116,11 @@ export default class PropertiesPlugin implements ITelemetryPlugin, IPropertiesPl
         this.context.applyOperatingSystemContxt(event);
         this.context.applyWebContext(event);
 
+        if (!event.data) {
+            event.data = {};
+        }
+        this.context.applyCustomProperties(event);
```
Comment (jpiyali) [inline vNext/extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts]: This is already done in the applicationinsights-analytics SKU. Why is this needed in properties plugin?

**Comment by hectorhdzg** [comment]
AI Analytics plugin currently expose telemetryInitializers to enhance all events, including adding customProperties to the event
